deploy/helm/charts/README.md

```diff
@@ -81,54 +81,54 @@ The following table lists the configurable parameters of the OpenEBS CStor chart
 | admissionServer.image.pullPolicy | string | `"IfNotPresent"` | Admission webhook image pull policy |
 | admissionServer.image.registry | string | `nil` | Admission webhook image registry |
 | admissionServer.image.repository | string | `"openebs/cstor-webhook"` | Admission webhook image repo |
-| admissionServer.image.tag | string | `"2.5.0"` | Admission webhook image tag |
+| admissionServer.image.tag | string | `"2.6.0"` | Admission webhook image tag |
 | admissionServer.nodeSelector | object | `{}` | Admission webhook pod node selector |
 | admissionServer.podAnnotations | object | `{}` |  Admission webhook pod annotations |
 | admissionServer.resources | object | `{}` | Admission webhook pod resources |
 | admissionServer.securityContext | object | `{}` | Admission webhook security context |
 | admissionServer.tolerations | list | `[]` | Admission webhook tolerations |
 | csiController.annotations | object | `{}` | CSI controller annotations |
 | csiController.attacher.image.pullPolicy | string | `"IfNotPresent"` | CSI attacher image pull policy |
-| csiController.attacher.image.registry | string | `"quay.io/"` |  CSI attacher image registry |
-| csiController.attacher.image.repository | string | `"k8scsi/csi-attacher"` |  CSI attacher image repo |
+| csiController.attacher.image.registry | string | `"k8s.gcr.io/"` |  CSI attacher image registry |
+| csiController.attacher.image.repository | string | `"sig-storage/csi-attacher"` |  CSI attacher image repo |
 | csiController.attacher.image.tag | string | `"v3.1.0"` | CSI attacher image tag |
 | csiController.attacher.name | string | `"csi-attacher"` |  CSI attacher container name|
 | csiController.componentName | string | `""` | CSI controller component name |
 | csiController.driverRegistrar.image.pullPolicy | string | `"IfNotPresent"` | CSI driver registrar image pull policy  |
-| csiController.driverRegistrar.image.registry | string | `"quay.io/"` | CSI driver registrar image registry |
-| csiController.driverRegistrar.image.repository | string | `"k8scsi/csi-cluster-driver-registrar"` | CSI driver registrar image repo |
+| csiController.driverRegistrar.image.registry | string | `"k8s.gcr.io/"` | CSI driver registrar image registry |
+| csiController.driverRegistrar.image.repository | string | `"sig-storage/csi-cluster-driver-registrar"` | CSI driver registrar image repo |
 | csiController.driverRegistrar.image.tag | string | `"v1.0.1"` |  CSI driver registrar image tag|
```

Comment:
Can we change this corresponding to the values.yaml
```suggestion
| csiController.driverRegistrar.image.tag | string | `"v2.1.0"` |  CSI driver registrar image tag|
```

Reply:
Done.